uiowa/uiowa
# Restyled RSS link to use a modified button style

Addresses #4695 and #4435

[See "Idea 1" on #4695 ](https://github.com/uiowa/uiowa/issues/4695#issuecomment-1022286910)

<img width="961" alt="Screen Shot 2022-01-28 at 9 32 39 AM" src="https://user-images.githubusercontent.com/472923/151575121-71403392-a491-4225-849e-2bd653b7ab6b.png">

@bspeare -- I had to adjust the button styles a little bit just to keep this button aligned with the content, consequently it loses its outline on hover and its left/right padding. Feel free to modify if needed.

# How to test

- Run `blt frontend`
- Check pages/views where the the feed_icons template is placed, e.g., taxonomy terms, article pages, etc.
